### DISCUSSION TITLE
[Feature Request] Party Rest and Loot options

### DISCUSSION THREAD

**Discussion by R0gu3P403n1x** [EddieDover/fvtt-party-sheet#32] [Ideas]
Including the ability to have the party sheet allow for tracking of group loot and rest options would be awesome.

**Comment by EddieDover** [comment]
While these would indeed be awesome to have, there's potential that they reach beyond the scope of intent for this module. 

For example, I could easily add a button and prompt for the GM that would let them forcibly make everyone in the party take a long or short rest, but that potentially would only work for 5E. Add to this the request that the prompt display for the player and they be allowed to choose to take a rest. Now the plugin would need to be a client-gm plugin and not just a gm plugin, and I'd have to add socket connectivity and network callbacks. You can see how things would get complicated quickly, and that's not even discussing passing around inventory items yet.

This module was intended to allow the GM to visualize data in a way they wanted, and while I'm open to suggestions for ways to improve it, they need to be ways that don't pollute the core intent: A fully customizable and system-agnostic way of visualizing party data.

If you're using 5E, I'd recommend the Rest Recovery 5E plugin: https://github.com/roth-michael/FoundryVTT-RestRecovery
It adds a new button to the player list for rest options:
![image](https://github.com/user-attachments/assets/21b1a54c-13c4-474b-ac1e-c6a754ac34b2)
I use it myself and it works great.

Unfortunately, I was using a fantastic 5E plugin that did loot management, loot splitting, and distribution after combat, but I don't remember the name and it hasn't been updated for V12 yet, so I'd have to dig around and find it, but I will if you're interested (and still on V11... and using 5E).

**Comment by EddieDover** [comment]
I'm going to close this discussion for now but feel free to continue your chat with me on discord. I'm absolutely open to creating a plugin, or helping port one over, that can handle the resting portion of your request for Pathfinder.
